docs/design/core/metrics/DecisionLog.md

```diff
@@ -0,0 +1,41 @@
+# Decision Log for SDK V2 Metrics
+
+Note: The decision log process was implemented late in this project, so decisions earlier than 7/30/20 are not included 
+below.
+
+## Log Entry Template
+
+**Source:** (Meeting/aside/pair programming discussion/daily standup) to (discuss/implement) X
+
+**Attendees:** bmaizels@, dongie@, ikors@, millem@, salande@, vinoddin@, zhouqua@, zoewang@
+
+**Closed Decisions:**
+
+1. Question? Decision. Justification.
+
+**Open Decisions:**
+
+1. (Old/Reopened/New) Question?
+
+## 7/30/20
```

Comment:
Should this be 6/30/20? Or it's not a date?

Reply:
+1 good catch! I guess I subconsciously really want 2020 to be over faster.